### PR TITLE
Added POA support

### PR DESCRIPTION
**Pull request by chinmay-browserstack**
PoA Support for appium java.
We will add PercyScreenshot for PoA which will take screenshot using automate.

### CODE REVIEW

**Comment by pankaj443** [comment]
@chinmay-browserstack we had to do some changes related to screenshot options as well right? add warnings if a particular option doesn’t belong to POA. Same would apply for dotnet PR as well

**Comment by pankaj443** [comment]
ok so from what i see, we have `Map<String, Object>` type for POA options and not ScreenshotOptions. if someone uses Percy class to trigger POA and uses ScreenshotOptions we throw not implemented and using `Map<String, Object>` type will forward to CLI?

**Comment by chinmay-browserstack** [comment]
> will forward to CLI?

Yes we will throw error and when options received in Map it is forwarded to CLI and will be handled there

**Review by pankaj443 — APPROVED** [review]

File: src/main/java/io/percy/appium/IPercy.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package io.percy.appium;
+
+import io.percy.appium.lib.ScreenshotOptions;
+
+import java.util.Map;
+
+public class IPercy {
```
Comment (ninadbstack) [inline src/main/java/io/percy/appium/IPercy.java:7]: [Optional] should mark abstract

**Review by ninadbstack — APPROVED** [review]
LGTM

**Pull request merged**